### DISCUSSION TITLE
Github Student Pack Redirecting To Pricing

### DISCUSSION THREAD

**Discussion by Akashmishra1209** [community/community#141298] [GitHub Education]
### Hi everyone,

When I click the button to get the Student Developer Pack, it redirects me to pricing. I Am Unable To Get Benefits.I Have Tried All Methods.Thanks In Advanced

**Comment by rabbive** [comment]
To access the GitHub Student Developer Pack, please visit https://education.github.com/pack/join . Follow the procedures outlined on the page, where you will be asked to provide proof of your student status. Make sure you meet the eligibility criteria, such as being at least 13 years old and currently enrolled as a student.

**Reply by Akashmishra1209** [reply]
When I Click On I Am A Studen It Is Again Redirecting Me To Pricing

**Comment by Akashmishra1209** [comment]
Dear Sir,
                   When I Click On I Am A Student,It Is Redirecting Me To
Pricing Section

**Comment by davevad93** [comment]
Hi @Akashmishra1209 ,

This has also happened to other users as you can see in these discussions, #131668 , #135388 , #135546, #136400. I suggest you to contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your issue.

**Comment by Akashmishra1209** [comment]
Thank you, I will do that.
